src/CMakeLists.txt

```diff
@@ -1,3 +1,18 @@
+OPTION(SC_SYSTEM_OWN_MSEED "Use included libmseed" ON)
+
+IF (NOT SC_SYSTEM_OWN_MSEED)
+	SET(MSEED_REQUESTED_VERSION 2)
+	FIND_PACKAGE(MSEED REQUIRED)
+	SET(LIBMSEED_INCLUDE_DIR ${MSEED_INCLUDE_DIR})
+	SET(LIBMSEED_LIBRARY ${MSEED_LIBRARY})
+ELSE (NOT SC_SYSTEM_OWN_MSEED)
+	SET(LIBMSEED_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/libs/3rd-party/libmseed CACHE INTERNAL "")
```

Comment:
The include dir must refer to ${CMAKE_CURRENT_SOURCE_DIR}/system/libs/3rd-party/libmseed